guide/creating_element/testing.md

```diff
@@ -144,6 +144,54 @@ end
 
 Once again, we are comparing the pipeline's output with our reference file to assert that everything works correctly.
 
+### Customizing pipeline module
+
+[`Membrane.Testing.Pipeline`](https://hexdocs.pm/membrane_core/Membrane.Testing.Pipeline.html) allows us to customize every callback of [`Membrane.Pipeline`](https://hexdocs.pm/membrane_core/Membrane.Pipeline.html). Let's create Pipeline that will answer to ping messages. To do this, we must define our custom module.
+
+```elixir
+defmodule MockPipeline do
```

Comment:
Not necessarily Mock. This feature may be used to test pipelines alongside with elements. Maybe `Example.Pipeline`?